Doc/library/sys.rst

```diff
@@ -1367,13 +1369,14 @@ always available.
 
 .. data:: version_info
 
-   A tuple containing the five components of the version number: *major*, *minor*,
-   *micro*, *releaselevel*, and *serial*.  All values except *releaselevel* are
-   integers; the release level is ``'alpha'``, ``'beta'``, ``'candidate'``, or
-   ``'final'``.  The ``version_info`` value corresponding to the Python version 2.0
-   is ``(2, 0, 0, 'final', 0)``.  The components can also be accessed by name,
-   so ``sys.version_info[0]`` is equivalent to ``sys.version_info.major``
-   and so on.
+   A :class:`~collections.namedtuple` containing the five components of the
```

Comment:
`version_info` is not a `collections.namedtuple`. It is a *named tuple*. `collections.namedtuple` implements named tuple interface, but not all named tuples are `collections.namedtuple`.

Just add the word "named" before "tuple".